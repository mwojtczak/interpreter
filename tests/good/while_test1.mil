function main () {
	x = 1;
	y = 0;
	while (x <= 10) {
		y = y + x;
		x = x + 1;
		print(x);
		print(y);
	};
	print(x);
	print(y);
}
