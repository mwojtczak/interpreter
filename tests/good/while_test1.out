2
1
3
3
4
6
5
10
6
15
7
21
8
28
9
36
10
45
11
55
11
55
